spec/ddtrace/contrib/delayed_job/plugin_spec.rb

```diff
@@ -7,13 +7,22 @@
 require_relative 'delayed_job_active_record'
 
 RSpec.describe Datadog::Contrib::DelayedJob::Plugin, :delayed_job_active_record do
-  let(:sample_job_object) { double('sample_job', perform: nil) }
-  let(:sample_job_class) { class_double('SampleJob', new: sample_job_object) }
+  class SampleJob
```

Comment:
I don't think we should replace these `let` blocks with constants. The constants are immutable between tests and can cause naming conflicts. I suspect the existing RSpec doubles might not cut it because the changes introduced might actually depend on reflect real constants, however, I think we could convert these blocks to dynamic constants instead which would cover both sets of needs.

Something like:

```ruby
let(:sample_job_object) do
  stub_const("SampleJob", Class.new do
    def perform; end
  end)
end
```